mozart/services/api_v02/tags.py

```diff
@@ -37,17 +37,17 @@ def put(self):
         tag = request_data.get('tag')
         app.logger.info('_id: %s\n _index: %s\n tag: %s' % (_id, _index, tag))
 
-        if _index != 'job_status-current':
-            app.logger.error('user tags only for index: job_status-current')
+        if _id is None or _index is None or tag is None:
             return {
                 'success': False,
-                'message': 'user tags only for index: job_status-current'
+                'message': 'id, index and tag must be supplied'
             }, 400
 
-        if _id is None or _index is None or tag is None:
+        if not _index.startswith("job_status-"):
+            app.logger.error('user tags only for index: job_status-*')
             return {
                 'success': False,
-                'message': 'id, index and tag must be supplied'
+                'message': 'user tags only for index: job_status-*'
             }, 400
 
         dataset = mozart_es.get_by_id(index=_index, id=_id, ignore=404)
```

Comment:
@DustinKLo: should this be updated to `search_by_id` too?

Reply:
@pymonger 
We’re okay using get_by_id here, in the hysds_ui component we’re passing in the document’s index so we don’t need to use the _search API
https://github.com/hysds/hysds_ui/blob/develop/src/components/FigaroDataViewer/index.jsx#L37